samples/snippets/snippets_test.py

```diff
@@ -201,15 +193,57 @@ def test_avg_query_property_filter(self, capsys, client):
     def test_multiple_aggregations_query(self, capsys, client):
         tasks = snippets.multiple_aggregations_query(client)
         captured = capsys.readouterr()
+        assert "avg_aggregation value is 3.0" in captured.out
+        assert "count_aggregation value is 3" in captured.out
+        assert "sum_aggregation value is 9" in captured.out
+        assert captured.err == ""
+
+        client.entities_to_delete.extend(tasks)
+
+    # @backoff.on_exception(backoff.expo, AssertionError, max_time=240)
```

Comment:
Should we uncomment these decorators?
```suggestion
    @backoff.on_exception(backoff.expo, AssertionError, max_time=240)
```

Reply:
good catch, don't think that was intentional